package.json

```diff
@@ -19,7 +19,7 @@
     "local": "yarn run testnet local",
     "testnet": "node tasks/testnet.js",
     "lint": "yarn lint:eslint && yarn lint:format",
-    "lint:eslint": "eslint -f ./node_modules/eslint-friendly-formatter '{,**/}*.{js,vue}' '!testArtifacts/**' '!app/networks/**'",
+    "lint:eslint": "eslint -f ./node_modules/eslint-friendly-formatter '{,**/}*.{js,vue}' '!app/networks/**'",
```

Comment:
should we add app/networks also to the eslintignore file?

Reply:
Done.